src/arm/BBORG_MOTOR-00A2.dts

```diff
@@ -9,135 +12,97 @@
+	P8_14_pinmux { status = "okay"; pinctrl-0 = <&P8_14_gpio_pin>; };	/* motor3 direction */
+	P8_16_pinmux { status = "okay"; pinctrl-0 = <&P8_16_gpio_pin>; };	/* motor2 direction */
+	P8_18_pinmux { status = "okay"; pinctrl-0 = <&P8_18_gpio_pin>; };	/* motor1 direction */
+	P8_19_pinmux { status = "okay"; pinctrl-0 = <&P8_19_pwm_pin>; };	/* motor4 speed */
+	P8_26_pinmux { status = "okay"; pinctrl-0 = <&P8_26_gpio_pin>; };	/* motor4 direction */
+	P9_14_pinmux { status = "okay"; pinctrl-0 = <&P9_14_pwm_pin>; };	/* motor2 speed */
+	P9_16_pinmux { status = "okay"; pinctrl-0 = <&P9_16_pwm_pin>; };	/* motor1 speed */
+};
 
-	/*
-	 * Free up the pins used by the cape from the pinmux helpers.
-	 */
-	fragment@1 {
-		target = <&ocp>;
-		__overlay__ {
-			P8_13_pinmux { status = "disabled"; };	/* m3.pwm2.b */
-			P8_14_pinmux { status = "disabled"; };	/* m3.high */
-			P8_16_pinmux { status = "disabled"; };  /* m2.high */
-			P8_18_pinmux { status = "disabled"; };	/* m1.high */
-			P8_19_pinmux { status = "disabled"; };	/* m4.pwm2.a */
-			P8_26_pinmux { status = "disabled"; };  /* m4.high */
-			P9_14_pinmux { status = "disabled"; };	/* m2.pwm1.a */
-			P9_16_pinmux { status = "disabled"; };	/* m1.pwm1.b */
-		};
-	};
+// Motor 1, 2
+&bone_pwmss_1 {
+	status = "okay";
+};
 
-	fragment@2 {
-		target = <&am33xx_pinmux>;
-		__overlay__ {
-			bborg_motor_led_pins: pinmux-bborg-proto-led-pins {
-				pinctrl-single,pins = <
-					AM33XX_IOPAD(0x0828, PIN_OUTPUT_PULLUP | INPUT_EN | MUX_MODE7)	/* P8_14: gpmc_ad10.gpio0_26 */
-					AM33XX_IOPAD(0x0838, PIN_OUTPUT_PULLUP | INPUT_EN | MUX_MODE7)	/* P8_16: gpmc_ad14.gpio1_14 */
-					AM33XX_IOPAD(0x088c, PIN_OUTPUT_PULLUP | INPUT_EN | MUX_MODE7)	/* P8_18: gpmc_clk.gpio2_1 */
-					AM33XX_IOPAD(0x087c, PIN_OUTPUT_PULLUP | INPUT_EN | MUX_MODE7)	/* P8_26: gpmc_csn0.gpio1_29 */
-				>;
-			};
+&bone_pwm_1 {
+	status = "okay";
+};
 
-			bborg_motor_ehrpwm1_pins: pinmux-bborg-proto-ehrpwm1-pins {
-				pinctrl-single,pins = <
-					AM33XX_IOPAD(0x0848, PIN_OUTPUT_PULLDOWN | MUX_MODE6)	/* P9_14: gpmc_a2.ehrpwm1a */
-					AM33XX_IOPAD(0x084c, PIN_OUTPUT_PULLDOWN | MUX_MODE6)	/* P9_16: gpmc_a3.ehrpwm1b */
-				>;
-			};
+// Motor 3, 4
+&bone_pwmss_2 {
+	status = "okay";
+};
 
-			bborg_motor_ehrpwm2_pins: pinmux-bborg-proto-ehrpwm2-pins {
-				pinctrl-single,pins = <
-					AM33XX_IOPAD(0x0824, PIN_OUTPUT_PULLDOWN | MUX_MODE4)	/* P8_13: gpmc_ad9.ehrpwm2b */
-					AM33XX_IOPAD(0x0820, PIN_OUTPUT_PULLDOWN | MUX_MODE4)	/* P8_19: gpmc_ad8.ehrpwm2a */
-				>;
-			};
-		};
-	};
+&bone_pwm_2 {
+	status = "okay";
+};
 
-	fragment@3 {
-		target = <&epwmss1>;
-		__overlay__ {
-			status = "okay";
-		};
-	};
 
-	fragment@4 {
-		target = <&ehrpwm1>;
-		__overlay__ {
-			pinctrl-names = "default";
-			pinctrl-0 = <&bborg_motor_ehrpwm1_pins>;
+/*
+ * Easy Motor direction control through sysfs (/sys/class/leds/motor*) using gpio-leds driver
+ * See these files for the led_P8_#/led_P9_#  definition
```

Comment:
Interesting to carry these led definitions at the "bus" level. I suppose it works if they are useful enough. We need a sane way to document helpers that are exposed by the "bus".

Please update [the wiki page specification](https://elinux.org/Beagleboard:BeagleBone_cape_interface_spec) with usage guidelines for what interfaces are exposed for overlays to use.